### PR TITLE
Add .pre-commit-hooks.yaml

### LINKED ISSUE #2: Add pre-commit hook
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Provide a pre-commit hook. Would you accept a PR for this?

**Describe alternatives you've considered**
None.

### PR DESCRIPTION
**Pull request by rpdelaney**
## Description

Adds `.pre-commit-hooks.yaml` to support use of this project as a [`pre-commit`](https://pre-commit.com) hook.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Related Issue

Closes #2.

## Motivation and Context

Adds support for using `anti-trojan-source` in pre-commit.

## How Has This Been Tested?

Here's what happens when I run the hook on this very repo:

```console
$ pre-commit try-repo ~/src/me/anti-trojan-source --verbose --all-files
[INFO] Initializing environment for ../../../Users/ryan/src/me/anti-trojan-source.
===============================================================================
Using config:
===============================================================================
repos:
-   repo: ../../../Users/ryan/src/me/anti-trojan-source
    rev: 8c1651b9dd8318fa754d47d567fc05913d91efd9
    hooks:    -   id: anti-trojan-source
===============================================================================
[INFO] Installing environment for ../../../Users/ryan/src/me/anti-trojan-source.
[INFO] Once installed this environment will be reused.
[INFO] This may take a few minutes...
Detect unicode bidi attacks..............................................Failed
- hook id: anti-trojan-source
- duration: 0.21s
- exit code: 1

[✓] No case of trojan source detected

[✓] No case of trojan source detected

[x] Detected cases of trojan source in the following files:
| 
 - README.md

[x] Detected cases of trojan source in the following files:
| 
 - .github/anti-trojan-source-logo.png

[x] Detected cases of trojan source in the following files:
| 
 - __tests__/__fixtures__/true-trojan-source.js

[x] Detected cases of trojan source in the following files:
| 
 - __tests__/__fixtures__/true-trojan-source-complex1.ts

[✓] No case of trojan source detected

[✓] No case of trojan source detected

[✓] No case of trojan source detected
```

## Screenshots (if appropriate):
N/A

## Checklist:

- [x] I have updated the documentation (if required).
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [x] I added a picture of a cute animal cause it's fun

yukicat

### CODE REVIEW

**Comment by lirantal** [comment]
Very cool @rpdelaney 
Thank you for sending the PR 👏👏👏

File: README.md [hunk 1]
```diff
@@ -103,6 +103,18 @@ const isDangerous = hasTrojanSource({
...
 `hasTrojanSource` returns a boolean.
 
+# Use as a pre-commit hook
+
+To add this tool to your project as a [`pre-commit`](https://pre-commit.com) hook, try this sample configuration in `.pre-commit-config.yaml`:
+
+```yaml
+repos:
+  - repo: https://github.com/lirantal/anti-trojan-source
+    rev: v1.3.3  # choose the release you want
```
Comment (lirantal) [inline README.md:113]: @rpdelaney I assume you'll need to bump this one to the new version once this PR is merged or will it work still with 1.3.3?

**Review by lirantal — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (rpdelaney) [reply README.md:113]: pre-commit will try to download whatever release the user enters here. You can update the readme with the latest version after every release if you like (or care to automate it), but if someone copy-pastes it should work (they'll just get an older version).

Comment (lirantal) [reply README.md:113]: Gotcha! Thanks for explaining :)

**Pull request merged**